tt_eager/tt_dnn/queue_api.hpp

```diff
@@ -0,0 +1,79 @@
+void EnqueueAllocateDeviceBuffer(Queue&, DeviceBuffer&);
+void EnqueueAllocateHostBuffer(Queue&, HostBuffer&);
```

Comment:
Do you need these enqueue allocate APIs? Is what we currently have for allocating memory sufficient?

Reply:
needed for Metal graph